codegen/render/kube_types_renderer.go

```diff
@@ -23,6 +23,13 @@ type KubeCodeRenderer struct {
 	// the relative path to the api dir
 	// types will render in the package <module>/<apiRoot>/<group>/<version>
 	ApiRoot string
+
+	ContribTemplates ContribTemplates
+}
+
+type ContribTemplates struct {
```

Comment:
Why are these all being called `Contrib`? RenderMatchers seems more appropriate here.

Reply:
This struct is meant to house all contributed templates

Reply:
Gotcha, I think once something has been added to skv2 we should make it a first class citizen. If we keep adding things to the `Contrib` struct it will get unruly and confusing.

Reply:
Wanted guidance on this very question—the code here was meant only as a placeholder in lieu of an actual structure for contributed templates